DataFormats/FWLite/test/test.cppunit.cpp

```diff
@@ -176,7 +176,6 @@ void testRefInROOT::testEventBase() {
       edm::Handle<edmtest::OtherThingCollection> pOthers;
       eventBase->getByLabel(tag, pOthers);
       CPPUNIT_ASSERT(pOthers.isValid());
-      pOthers->size();
```

Comment:
Or how about
```cpp
      CPPUNIT_ASSERT(pOthers->size() > 0);
```
?

Reply:
sounds reasonable @makortel. By the way, can we have valid `edm::Handle` but with `0` size? If yes then are we sure that in this unit test size shiould be > 0?

Reply:
> By the way, can we have valid `edm::Handle` but with `0` size?

It can, the size is a property of the data product, and `Handle` cares only on the existence of the product, but not of its internal details.

> If yes then are we sure that in this unit test size shiould be > 0?

As far as I can tell, this block (with `tag`) accesses the same product as the block above (with `tagFull`) which does a similar assertion.

Reply:
thanks, I have updated the code as suggested [here](https://github.com/cms-sw/cmssw/pull/39522#discussion_r982464441)